Author: Pino Toscano <pino@debian.org>
Description: Quick fix to compile konsole on Hurd
 Temporarly define MAXPATHLEN if not defined already (e.g. on Hurd).
Last-Update: 2013-07-13
Forwarded: not-needed

Index: konsole/src/ProcessInfo.cpp
===================================================================
--- konsole.orig/src/ProcessInfo.cpp	2013-09-26 13:03:44.354665475 +0000
+++ konsole/src/ProcessInfo.cpp	2013-09-26 13:03:44.350665627 +0000
@@ -61,6 +61,10 @@
 #   endif
 #endif
 
+#ifndef MAXPATHLEN
+#define MAXPATHLEN 4096
+#endif
+
 using namespace Konsole;
 
 ProcessInfo::ProcessInfo(int aPid , bool enableEnvironmentRead)
